.github/workflows/workflow.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
           cd build
           ctest --verbose
       - name: Upload Artifact
-        if: startsWith(github.ref, 'refs/tags/') || github.ref_name == 'main'
+#        if: startsWith(github.ref, 'refs/tags/') || github.ref_name == 'main'
```

Comment:
I disabled this line such that the user at https://github.com/dethrace-labs/dethrace/issues/330 can download a built binary.